### DISCUSSION TITLE
wait-start after installing velociraptor from wes gitlab

### DISCUSSION THREAD

**Discussion by marianoka** [Security-Onion-Solutions/securityonion#4092] [General] [answered]
Hi Pals!
I have installed Velociraptor server at the onion vm, and installation goes great. But afeter rebooting VM, only 3 services starts.
If you doy manually, it works. What should I do to fix this issue.
Thanks
Mariano

**Comment by weslambert** [comment, marked as answer]
Hi @marianoka ,

If you have questions with regard to the aforementioned integration, please open an issue here: https://github.com/weslambert/securityonion-velociraptor

**Reply by palichx** [reply]
After installation "securityonion-velociraptor" so-nginx is down on all my nodes.  there is error in /opt/so/log/nginx/error.log
2021/05/05 13:29:45 [emerg] 1#1: no host in upstream ":8080" in /etc/nginx/nginx.conf:48
